setup.py

```diff
@@ -1,25 +1,25 @@
-from setuptools import setup, find_packages
-
-setup(name='pysumma',
-      version='2.0.0',
-      description='an Object-Oriented Python wrapper for SUMMA model',
-      url='https://github.com/uva-hydroinformatics/pysumma.git',
-      author='YoungDon Choi',
-      author_email='choiyd1115@gmail.com',
-      license='MIT',
-      packages=find_packages(),
-      install_requires=[
-          'numpy',
-          'xarray>0.10.9',
-          'matplotlib',
-          'geopandas',
-          'shapely',
-          'seaborn',
-          'pandas>=0.25',
-          'hs_restclient==1.3.4',
-          'distributed',
-          'fiona',
-          'netcdf4'
-          ],
-      include_package_data=True,
-      test_suite='pysumma.tests')
\ No newline at end of file
+from setuptools import setup, find_packages
+
+setup(name='pysumma',
+      version='2.0.0',
+      description='A python wrapper for SUMMA',
+      url='https://github.com/UW-Hydro/pysumma.git',
+      author='YoungDon Choi, Andrew Bennett',
+      author_email='choiyd1115@gmail.com, andrbenn@uw.edu',
+      license='MIT',
+      packages=find_packages(),
+      install_requires=[
+          'numpy',
+          'xarray>0.10.9',
+          'matplotlib',
+          'geopandas',
+          'shapely',
+          'seaborn',
+          'pandas>=0.25',
+          'distributed',
```

Comment:
I think we still need hs_restclient to download and upload HydroShare resources.

Reply:
Fixed this